src/alpaca_typer.erl

```diff
@@ -2054,7 +2053,7 @@ typ_apply(Env, Lvl, TypF, NextVar, Args, Line) ->
                     {error, _}=Err -> Err;
                     {Typ, NV} ->
                         case get_cell(Typ) of
-                            {t_receiver, Recv2, RetTyp} ->
+                            {t_receiver, _Recv2, RetTyp} ->
```

Comment:
I think it can be safely replaced with `_`.  IIIRC the received messages cells will all have unified prior to this so it's not necessary for us to keep a reference to this particular cell when `R2` will contain the same type or link to it.  The pattern match here, from what I recall, is in case the function application itself returns another receiver that needs to be unified with the enclosing one.